experimental-examples/kitchen-sink/tina/custom-component-reference-select/reference-field-schema.tsx

```diff
@@ -49,6 +26,23 @@ const referenceField = {
           return s.path
       }
     },
+    //Static example - user can define which field and value they want to filter
+    // collectionFilter: {
+    //   author: {
+    //     location: 'melbourne',
+    //   },
+    //   post: {
+    //     title: 'hello world',
+    //   },
+    // },
+    //Dynamic example - user can define the function they want here but need to make sure the return type match the schema
+    collectionFilter: () => {
+      return {
+        author: {
+          location: getLocationFromHost('ndcsydney.com'),
+        },
+      }
+    },
```

Comment:
- Can we make this function inline to make this a little clearer?

Reply:
> Can we make this function inline to make this a little clearer?
Isn't this already inline ?

> Can we add another field that shows off the static example?
Intentionally leave the commented out code above in the same reference field so that we can have a clearer comparison on both method.

Reply:
> Isn't this already inline ?

No - I would move the function that is being called from another file into this one to make it a bit easier to see at a first glance. As per Matt's recommendation this morning 

> Intentionally leave the commented out code above in the same reference field so that we can have a clearer comparison on both method.

I guess that works - Would do 2 separate examples then on the docs page 🙏